testsuite/cluster-test/src/experiments/reconfiguration_test.rs

```diff
@@ -141,24 +190,37 @@ impl Experiment for Reconfiguration {
                 vec![update_txn],
             )
             .await?;
-            info!("Epoch {} committed", self.count + 1);
+            expect_epoch(&full_node_client, version, self.count + 1).await?;
+        }
+        let elapsed = timer.elapsed();
+        if let Some(job) = emit_job {
+            let stats = context.tx_emitter.stop_job(job).await;
+            context
+                .report
+                .report_txn_stats(self.to_string(), stats, elapsed, "");
+        } else {
+            context.report.report_text(format!(
+                "{} finished in {} seconds",
+                self.to_string(),
+                elapsed.as_secs()
+            ));
         }
 
         Ok(())
     }
 
     fn deadline(&self) -> Duration {
         // allow each epoch to take 20 secs
-        Duration::from_secs(self.count as u64 * 20)
+        Duration::from_secs(self.count as u64 * 10)
     }
 }
 
 impl fmt::Display for Reconfiguration {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         write!(
             f,
-            "Affected peer: {}, pod_name: {}, total epoch: {}",
-            self.affected_peer_id, self.affected_pod_name, self.count
+            "Reconfiguration[{}]: total epoch: {}",
```

Comment:
I think we probably don't need this `[{self.affected_pod_name}]`. It will appear in changelog, and probably will just pollute it.